### PR TITLE
add release command

### PR DESCRIPTION
**Pull request by meahow**

### CODE REVIEW

File: src/commands/release.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+description: "Release build release to PyPi using twine"
```
Comment (StrikerRUS) [inline src/commands/release.yml]:
```suggestion
description: "Release build release to PyPI using twine"
```

Comment (StrikerRUS) [inline src/commands/release.yml]:
```suggestion
description: "Upload build release to PyPi using twine"
```

File: src/commands/release.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+steps:
+  - run:
+      name: "Release to PyPi"
```
Comment (StrikerRUS) [inline src/commands/release.yml]:
```suggestion
      name: "Release to PyPI"
```

**Comment by felicianotech** [comment]
Thank you for contributing. This PR will need further review by CircleCI. This PR is eligible for Hacktoberfest/Orbtoberfest.

**Comment by meahow** [comment]
@StrikerRUS thanks for your comments
I applied your suggestions

**Comment by felicianotech** [comment]
We can't accept this PR at the moment. This PR relies on a command that this orb doesn't install nor is available in the `cimg/python` Convenience Image. This can create an issue for new users using the orb and trying to use a command that is not available.

**Pull request closed without merging**